test/DADBridgeWrapper.ts

```diff
@@ -0,0 +1,73 @@
+import Contracts from '../components/Contracts';
+import { TestERC20Token } from '../typechain';
+import { DADBridgeWrapper } from '../typechain';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { expect } from 'chai';
+import { utils, BigNumber } from 'ethers';
+import { ethers } from 'hardhat';
+
+const ROLE_ADMIN = utils.id('ROLE_ADMIN');
+const TOTAL_SUPPLY = BigNumber.from(1_000_000_000);
+const MINT_AMOUNT = BigNumber.from(1_000_000);
+
+describe('DADBridgeWrapper', () => {
```

Comment:
Note: since the current implementation is slightly incorrect (two tokens instead of a single one) - I'll go over the tests after it's pushed

Reply:
Done.